### DISCUSSION TITLE
An error occurred during installation.

### DISCUSSION THREAD

**Discussion by danielb2** [openzfs/zfs#15664] [General]
I'm getting the below when trying to install. I'm not seeing any log where I can try to debug.

<img width="461" alt="Screen Shot 2023-12-11 at 15 01 31" src="https://github.com/openzfs/zfs/assets/26678/aad3db23-86b6-4656-ae6c-27a43dfe85e5">

### The installation failed

The Installer encountered an error that caused the
installation to fail. Contact the software manufacturer
for assistance.

**Comment by rincebrain** [comment]
You probably wanted https://github.com/openzfsonosx/openzfs at the moment.

**Reply by danielb2** [reply]
yes, thank you

**Comment by danielb2** [comment]
@scineram what's your problem? I'm asking for help on how to debug this
